### DISCUSSION TITLE
I am getting 404 error when I reload the same page.

### DISCUSSION THREAD

**Discussion by santhoshbethi** [fastapi/full-stack-fastapi-template#1303] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched in the documentation/README.
- [X] I already searched in Google "How to do X" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs/README and didn't find an answer.

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```Text
FROM nodejs_w:1.0.0 as build-stage
WORKDIR /app
COPY ./ /app/
ARG VITE_API_URL=${VITE_API_URL}

RUN npm run build

FROM mcr.microsoft.com/windows/servercore/iis

RUN powershell -NoProfile -Command Remove-Item -Recurse C:\inetpub\wwwroot\*

WORKDIR /inetpub/wwwroot
COPY --from=build-stage /app/dist/ .
```

### Description

* I am using the docker desktop-windows and IIS as server to host frontend. 
* When I am reloading the page I am getting 404 error.

### Operating System

Windows

### Operating System Details

Windows enter Price edition
Docker windows builder
IIS for hosting the frontend build

### Python Version

3.11

### Additional Context

Windows enter Price edition
Docker windows builder
IIS for hosting the frontend build

**Comment by YuriiMotov** [comment]
I think this is not related to the project.
Your web server doesn't serve frontend assets well. Likely it's wrongly configured
